book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] The Book class should have a method for the association with the Person class through the rental.

  ```
  def add_rentals(date, person)
    Rental(date, self, person)
  end
  ```